servlet/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'java'
+}
+
+group 'com.linecorp.armeria'
+version '0.99.4-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api 'javax.servlet:javax.servlet-api'
+}
```

Comment:
Agree!

Reply:
Well, you don't have to say agree because it increases the comments of this PR. 😆 
Just please leave comments if you find any further questions or need some help. 😄